components/search/SearchResultsGrid.vue

```diff
@@ -24,11 +24,24 @@
       ContentCard
     },
     props: {
```

Comment:
perhaps we'll want to pass as a prop the maximum number of columns.
Then we could pass this in from the search/entity page and set 'card-deck-3-cols/card-deck-4-cols' accordingly.

Reply:
@mirjamverloop what do you think about this suggestion?

Reply:
After discussion, we will not do this here, but come back to it after this and #210 are into master.